base/cvd/cuttlefish/common/libs/utils/files.h

```diff
@@ -51,6 +51,11 @@ Result<void> MoveDirectoryContents(const std::string& source,
                                    const std::string& destination);
 bool FileHasContent(const std::string& path);
 Result<std::vector<std::string>> DirectoryContents(const std::string& path);
+Result<std::vector<std::string>> DirectoryContentsPaths(
+    const std::string& path);
+// does not include self-reference or parent reference ("." or "..")
+Result<std::vector<std::string>> DirectoryContentsByModTimeDesc(
```

Comment:
Does `DirectoryContentsByModTimeDesc` return the absolute paths similar to `DirectoryContentsPaths` or just the base name similar to `DirectoryContents`? If returning the paths, should we name it `DirectoryContentsPathsByModTimeDesc` for consistency?